frontend/javascripts/types/validation.ts

```diff
@@ -73,10 +73,9 @@ export const validateUrlStateJSON = (value: string) => {
   const json = validateWithSchemaSync("types::UrlManagerState", value);
   return _.cloneDeepWith(json, (value, key) => {
     if (key === "mappingType") {
-      if (value === "JSON" || value === null) {
-        return value;
-      }
-      return "HDF5";
+      if (value == null) return "HDF5";
```

Comment:
Is this intentional? Why should a mappingType of null correspond to hdf5? :thinking:

Reply:
ah thanks for the catch. My mistake ... wrong thinking from my side ...